code/API_definitions/geofencing.yaml

```
@@ -148,6 +148,9 @@ paths:
           $ref: "#/components/responses/Generic500"
         "503":
           $ref: "#/components/responses/Generic503"
+      security:
+        - openId:
+            - geofencing-subscriptions:write
```

```suggestion
            - geofencing:subscriptions:write
```
it is api-name:resource:action